app/main.py

```diff
@@ -1,3 +1,26 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as file:
+        data = json.load(file)
+
+    fuel_price: float = data["FUEL_PRICE"]
+    customers = [Customer(customer) for customer in data.get("customers")]
+    shops = [Shop(shop) for shop in data.get("shops")]

```

Comment:
Ensure that the `data` dictionary from the JSON file contains the keys 'customers' and 'shops', and that their values are in the expected format. This is crucial for the correct initialization of `Customer` and `Shop` instances.